Makefile

```diff
@@ -320,8 +320,7 @@ celeryd:
 	@if [ "$(VENV_BASE)" ]; then \
 		. $(VENV_BASE)/awx/bin/activate; \
 	fi; \
-	$(PYTHON) manage.py celeryd -l DEBUG -B -Ofair --autoreload --autoscale=100,4 --schedule=$(CELERY_SCHEDULE_FILE) -Q tower_scheduler,tower_broadcast_all,$(COMPOSE_HOST),$(AWX_GROUP_QUEUES) -n celery@$(COMPOSE_HOST)
```

Comment:
I noticed `tower_scheduler` is removed. What queue/routing key is responsible for signaling of `tower_scheduler.job.*` for signaling `run_job_launch` and `run_job_complete` ?

Reply:
I guess I'm wondering *where* that is configured now? I missed where it might be configured I think.

I had originally intended this to be separate from the `tower` queue, but I reckon it *could* be run on just the `tower` queue.

Reply:
The tower scheduler heartbeat was being run on the `tower` queue. The event-triggered (job launch, job finish) was executed on the special `tower_scheduler` queue.